web/client-api/client-api.gradle

```diff
@@ -135,6 +121,28 @@ def stopSelenium = project.tasks.register('stopSelenium', DockerRemoveContainer)
     task.removeVolumes.set true
 }
 
+def gwtUnitTest = tasks.register('gwtUnitTest', Test) { t ->
+    t.dependsOn(deephavenDocker.portTask, seleniumHealthy)
+    t.finalizedBy(deephavenDocker.endTask, stopSelenium)
```

Comment:
(note that you also have `t.finalizedBy(dhDocker.endTask)` below)

if you make this change, there is no longer a reason to have unit vs integration tests. instead, i would suggest removing the deephavenDocker dependencies, so that this only depends on selenium - it will make the unit tests somewhat faster (probably approx 2x) to run after any changes to the js api.